cmd/simulator/config/flags.go

```diff
@@ -59,6 +70,14 @@ func BuildConfig(v *viper.Viper) (Config, error) {
 		Timeout:      v.GetDuration(TimeoutKey),
 		BatchSize:    v.GetUint64(BatchSizeKey),
 		MetricsPort:  v.GetUint64(MetricsPortKey),
+		TestType:     v.GetString(TestTypeKey),
+	}
+	if endpointsFile := v.GetString(EndpointsFileKey); endpointsFile != "" {
```

Comment:
do we expect to have `end-points-file` and `end-points` flags at the same time? I feel like `end-points-file` should be sufficient itself.